packages/astro/src/content/types-generator.ts

```diff
@@ -362,6 +362,28 @@ function invalidateVirtualMod(viteServer: ViteDevServer) {
 	viteServer.moduleGraph.invalidateModule(virtualMod);
 }
 
+/**
+ * Takes a configPath and returns a normalized relative version:
+ *  -   If is not relative, it adds `./` to the beginning.
+ *  -   If it ends with `.ts`, it removes the extension.
+ *  -   It adds `.js` to the end.
+ *  -   It stringifies the result (adds `""` around it).
+ * @param from Config path from path.
+ * @param to Config path to path.
+ * @returns Normalized config path.
+ */
+function normalizeConfigPath(from: string, to: string) {
+	const normalizedConfigPath = path.relative(from, to);
```

Comment:
nit: I would call this variable `relativeConfigPath`

Reply:
I used "normalize" because the function is an "action" and the result would be the "relative" path. Maybe something like `configPathToRelative`?